### DISCUSSION TITLE
Why use the term "ability" and not "procedure"?

### DISCUSSION THREAD

**Discussion by Tudyx** [mitre/caldera#2760] [Q&A]
If I look at the definition of a procedure in the [MITRE FAQ](https://attack.mitre.org/resources/faq/), it's said that it's an implementation of a technique, which seems that's also what an ability stand for.

Is there any reasons to have both terms?

**Comment by citruswhale** [comment]
I may be wrong but I have a feeling that "procedure" refers to consecutive usage of the techniques. As in its chronological and it is made stepwise. However, "abilities" refers to which tactics the adversary is armed with/ specializes in
